fuel/src/test/kotlin/com/github/kittinunf/fuel/RequestValidationTest.kt

```diff
@@ -1,112 +1,106 @@
 package com.github.kittinunf.fuel
 
-import com.github.kittinunf.fuel.core.*
+import com.github.kittinunf.fuel.core.FuelManager
+import com.github.kittinunf.fuel.core.Method
 import com.github.kittinunf.fuel.core.interceptors.validatorResponseInterceptor
-import com.github.kittinunf.result.Result
-import com.github.kittinunf.result.getAs
 import org.hamcrest.CoreMatchers.notNullValue
 import org.hamcrest.CoreMatchers.nullValue
 import org.junit.Assert.assertThat
 import org.junit.Test
 import org.hamcrest.CoreMatchers.`is` as isEqualTo
 
-class RequestValidationTest : BaseTestCase() {
-    private val manager: FuelManager by lazy {
-        FuelManager().apply {
-            basePath = "http://httpbin.org"
-        }
-    }
-
+class RequestValidationTest : MockHttpTestCase() {
     @Test
     fun httpValidationWithDefaultCase() {
-        val preDefinedStatusCode = 418
+        // Register all valid
+        for (status in (200..299)) {
+            mockChain(
+                request = mockRequest().withMethod(Method.GET.value).withPath("/$status"),
+                response = mockResponse().withStatusCode(status)
+            )
+        }
+
+        // Register teapot
```

Comment:
sorry what is a teapot? (as and English guy I never thought I'd ask that)

Reply:
Status 418 is "I am a teapot" 

https://httpstatuses.com/418
https://tools.ietf.org/id/draft-nottingham-thanks-larry-00.html

It was used in the test before :)

Reply:
thanks for that